examples/api/lib/material/menu_bar/menu_bar.1.dart

```diff
@@ -0,0 +1,172 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Example for MenuBar.adaptive.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() {
+  runApp(const MaterialApp(
+    title: 'Adaptive Menu Sample',
+    home: Material(child: Home()),
+  ));
+}
+
+enum MenuSelection {
+  edit,
+  cut,
+  copy,
+  paste,
+  file,
+  open,
+  quit,
+  save,
+  saveAs,
+}
+
+String getLabel(MenuSelection selection) {
+  // Use a switch so that the analyzer will warn us if we aren't handling a
+  // case.
+  switch (selection) {
+    case MenuSelection.edit:
+      return 'Edit';
+    case MenuSelection.cut:
+      return 'Cut';
+    case MenuSelection.copy:
+      return 'Copy';
+    case MenuSelection.paste:
+      return 'Paste';
+    case MenuSelection.file:
+      return 'File';
+    case MenuSelection.open:
+      return 'Open';
+    case MenuSelection.quit:
+      return 'Quit';
+    case MenuSelection.save:
+      return 'Save';
+    case MenuSelection.saveAs:
+      return 'Save As...';
+  }
+}
```

Comment:
I guess you can put that inside the enum now? 😛

Reply:
Ooh! Good point. Still getting used to all this fancy stuff.  I remember when I had to code my own enum classes in Dart. :-)